Makefile

```diff
@@ -1,5 +1,7 @@
+# Y - in response to Would you like to add Slim Template Engine(yes/no)
+
 create:
-	rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
+	printf "Y\n" | rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
```

Comment:
Same with this [comment](https://github.com/nimblehq/rails-templates/pull/239#discussion_r501496294)